### PR TITLE
[Instrumentation.MySql] fix sql text incomplete

### PR DESCRIPTION
**Pull request by chad122**
Fixes #.

## Changes

Fix mysql incomplete db.statement.

Should use QueryNormalized event when it exists. Because cmdText in QueryOpened event is incomplete when cmdText.length>300.

### CODE REVIEW

File: src/OpenTelemetry.Instrumentation.MySqlData/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## Unreleased
+* Fix incomplete db.statement when the length>300
```
Comment (Kielek) [inline src/OpenTelemetry.Instrumentation.MySqlData/CHANGELOG.md:4]:
```suggestion
* Fix incomplete db.statement when the length>300
  [(#424)](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/424)
```

**Comment by chad122** [comment]
Who is the code owner? Please review the PR.

**Comment by utpilla** [comment]
> Who is the code owner? Please review the PR.

@moonheart Could you please review this PR?

**Comment by moonheart** [comment]
@chad122  Thanks for your awesome PR, LGTM!
The root cause is I missed the normalized query in 
https://github.com/mysql/mysql-connector-net/blob/0b13574eafb1f8671d9697f3e07a819eaacb1f22/MySQL.Data/src/TracingDriver.cs#L65-L85

**Comment by chad122** [comment]
> @chad122 Thanks for your awesome PR, LGTM! The root cause is I missed the normalized query in https://github.com/mysql/mysql-connector-net/blob/0b13574eafb1f8671d9697f3e07a819eaacb1f22/MySQL.Data/src/TracingDriver.cs#L65-L85

Yes, correct.

**Comment by utpilla** [comment]
Thanks @moonheart for the review and thanks @chad122 for creating this PR!

**Review by utpilla — APPROVED** [review]

**Comment by utpilla** [comment]
@chad122 Please create a PR to update the CHANGELOG if you would like a new version released for this package.

**Comment by chad122** [comment]
> @chad122 Please create a PR to update the CHANGELOG if you would like a new version released for this package.

OK.

**Pull request merged**